python/lsst/meas/algorithms/installGaussianPsf.py

```diff
@@ -76,6 +76,9 @@ def run(self, exposure):
         if exposure.hasPsf():
             psfModel = exposure.getPsf()
             psfSigma = psfModel.computeShape(psfModel.getAveragePosition()).getDeterminantRadius()
+            if math.isnan(psfSigma):
+                raise RuntimeError("Cannot create a Gaussian PSF model. "
+                                   "The computed exposure psfSigma is NaN!")
```

Comment:
This is ok, but what we really need is a new `AlgorithmError` in `psfexDeterminerTask` to raise if the fitted psf model is bad. Erfan added `PsfexTooFewGoodStarsError`, so you can crib from that.